mopidy_mpd/translator.py

```diff
@@ -18,7 +18,7 @@ def normalize_path(path, relative=False):
     return "/".join(parts)
 
 
-def track_to_mpd_format(track, position=None, stream_title=None):
+def track_to_mpd_format(track, position=None, stream_title=None, tagtypes=None):
```

Comment:
Does `tagtypes` need to be optional?

Reply:
To be clearer, I don't think this new argument should be optional.

Reply:
To be clear, my concern with this as optional was that if there was anywhere in this where we'd forgotten to supply `tagtypes`, that would be a bug.